README.md

```diff
@@ -117,6 +117,17 @@ ANVIL_IP_ADDR=0.0.0.0 anvil \
   --timestamp 1577836800
 ```
 
+### Profiling
+
+The most important binaries support [tokio-console](https://github.com/tokio-rs/console) to allow you a could look inside the tokio runtime.
+
+Simply enable the feature by passing `--enable-tokio-console true` when running a binary and then in another shell, run
```

Comment:
Actually specifically because of that we have to pass the bool as well:
```
cargo run -p orderbook -- --enable-tokio-console                                                             ─╯
    Finished `dev` profile [unoptimized + debuginfo] target(s) in 0.31s
     Running `target/debug/orderbook --enable-tokio-console`
error: a value is required for '--enable-tokio-console <ENABLE_TOKIO_CONSOLE>' but none was supplied
  [possible values: true, false]

For more information, try '--help'.
```

This was done because we configure our infra mostly via ENV variables and without that there was no way to disable a bool flag that had `true` as the default.

Reply:
Oh, I understand, got it! thanks!